src/index.ts

```diff
@@ -1008,15 +1026,20 @@ export class BlitzOptions {
             balance.id
         )
         const response = await this.wsApiClient.doRequest<BinaryOptionsOptionV1>(request)
-        return new BlitzOptionsOption(response)
+        const positions = await this.positionsByBalance(balance)
+        return new BlitzOptionsOption(response, positions)
     }
 
     /**
      * Returns blitz options positions facade class for specified balance.
      * @param balance - User's balance for which the positions are being requested.
      */
     public positionsByBalance(balance: Balance): Promise<Positions> {
-        return Positions.create(this.wsApiClient, 'blitz-option', balance)
+        const key = `${balance.id}-${balance.userId}`
```

Comment:
Key must contain instrument type, because two positions with different instrument types can have same option (external) id.
User id is redundant for key. You can remove it from key. Instrument type and balance id will be enough.